config.js

```diff
@@ -56,7 +56,7 @@ module.exports = {
     healthCheckUrl: process.env.NOTIFY_HEALTH_CHECK_URL || 'https://api.notifications.service.gov.uk/_status',
   },
   cmdApi: {
-    url: get('CMD_API_URL', 'http://localhost:9191'),
+    url: get('CMD_API_URL', 'http://localhost:8080'),
```

Comment:
Is this the port you have configured docker to run on in cmd-api?

Reply:
yes, I can show you today if you want, the UI uses 3005, so I was going to run the APIs on 8080 and 8081 for the csr api